ci_framework/roles/edpm_build_images/README.md

```diff
@@ -0,0 +1,35 @@
+## Role: edpm_build_images
```

Comment:
```suggestion
# edpm_build_images
```

Please also remove one level of `#` in the following titles in order to match the current doc format.